README.md

```diff
@@ -53,9 +59,8 @@ Major Changes From Original Version
 
 Updating to new FIT SDK versions
 --------------------------------
-Download the latest SDK from http://www.thisisant.com/pages/products/fit-sdk
 
-Update the profile:
+Download the latest [ANT FIT SDK](http://www.thisisant.com/resources/fit) and update the profile:
```

Comment:
I like that you updated the link but I think the formatting was better before when the two steps were separate.